edx_django_utils/private_utils.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
     """
     declared_middleware = getattr(settings, 'MIDDLEWARE', None)
     if declared_middleware is None:
-        declared_middleware = settings.MIDDLEWARE_CLASSES  # Django 1.8 support
+        declared_middleware = settings.MIDDLEWARE_CLASSES  # Pre-Django 2 support
```

Comment:
Does this mean it is backward compatible?  If so, why bump the version to `2.0.0`?

Reply:
It is backwards compatible through 1.10. This code was previously trying to be forward compatible, but I don't think it ever was.

I can update the docs to use new-style only, sure. Both are fine, and old-style will only go away once we stop supporting django 1.x. But since we are all on 1.11, new style is recommended path, so docs should push it.

Reply:
Oh, and I bumped the version to `2.0.0` because I'm dropping support for django 1.8